### ISSUE TITLE
Form - Provide alternative text for images - (2036570478)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5628]
### Module: 
 40 Form 

### Violation: 
 Provide alternative text for images 

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html) 

### Instance ID: 
 2036570478 

### Severity: 
 10 

### Description: 
 [Issue]
There are images after "What is the most important aspect(s) of a component?" that serve a decorative purpose but are exposed to screen readers.

[User Impact]
Screen readers may announce SVGs with no accessible name as "group." This gives screen reader users irrelevant and confusing information.

[Code Reference] 
 ```
<svg class="check-svg" viewbox="0 0 16 16"><path d=""></path></svg>
 ``` 

### Note: 
 [Suggestion]
Provide alternative text for images. Non-meaningful images must be marked as decorative, which causes screen readers to ignore them. To mark an ``` <svg> ``` element as decorative, add aria-hidden="true" to the element.

[Compliant Code Example] 
 ```
<svg aria-hidden="true" class="check-svg" viewbox="0 0 16 16"><path d=""></path></svg>
 ```
 

### Media Type: 
 Images 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Fixed / mitigated in #5325 (`beta.94`).
